src/emulator/auth/state.ts

```diff
@@ -714,32 +729,24 @@ export class TenantProjectState extends ProjectState {
     return this._tenantConfig;
   }
 
-  // TODO(lisajian): Handle divergence in tenant config settings between what is
-  // needed for admin SDK (default disabled, parallels production) vs emulator
-  // tests (default enabled, for convenience)
   get allowPasswordSignup() {
-    return this._tenantConfig.allowPasswordSignup ?? true;
+    return this._tenantConfig.allowPasswordSignup!;
```

Comment:
What this compiles down to is `return this._tenantConfig.allowPasswordSignup`. The `!` implies the value will always be present, which is fine, but if something goes haywire the result is that undefineds or nulls will be passed back. Since undefined/null are falsey, I think the strictly correct behavior here should be `return this._tenantConfig.allowPasswordSignup || false`. WDYT?

Reply:
Oh man, sorry - I was mulling over this bang operator last night / this morning, just published a new revision that makes these fields required, thanks for taking a look!!